From 3d2afbf599f4fb65d6707cafaebfb55f087996db Mon Sep 17 00:00:00 2001
From: Vision Systems GmbH <contact@visionsystems.de>
Date: Thu, 28 Aug 2014 11:19:43 +0200
Subject: [PATCH 1/2] Ignore write-protect flag

---
 drivers/scsi/sd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/scsi/sd.c b/drivers/scsi/sd.c
index aef3f8f..7cbff90 100644
--- a/drivers/scsi/sd.c
+++ b/drivers/scsi/sd.c
@@ -2000,7 +2000,7 @@ sd_read_write_protect_flag(struct scsi_disk *sdkp, unsigned char *buffer)
 		sd_printk(KERN_WARNING, sdkp,
 			  "Test WP failed, assume Write Enabled\n");
 	} else {
-		sdkp->write_prot = ((data.device_specific & 0x80) != 0);
+		sdkp->write_prot = 0;
 		set_disk_ro(sdkp->disk, sdkp->write_prot);
 		if (sdkp->first_scan || old_wp != sdkp->write_prot) {
 			sd_printk(KERN_NOTICE, sdkp, "Write Protect is %s\n",
-- 
1.7.7

